src/transaction/log_manager.c

```diff
@@ -7705,6 +7691,13 @@ log_tran_do_postpone (THREAD_ENTRY * thread_p, LOG_TDES * tdes)
   assert (tdes->topops.last < 0);
 
   log_append_commit_postpone (thread_p, tdes, &tdes->posp_nxlsa);
+
+  if (tdes->m_log_postpone_cache.do_postpone (*thread_p, &tdes->posp_nxlsa))
+    {
+      // do postpone from cache first
+      return;
```

Comment:
what about adding two stats to count the number of postpone operations that executed with and without cache?
These might help us to tune the size of cache.

Reply:
A jira issue was open - http://jira.cubrid.org/browse/CBRD-23215

Reply:
sorry I didn't notice the issue you opened.